bundle/manifests/sonataflow-operator-controllers-config_v1_configmap.yaml

```diff
@@ -29,6 +29,18 @@ data:
     sonataFlowDevModeImageTag: ""
     # The default name of the builder configMap in the operator's namespace
     builderConfigMapName: "sonataflow-operator-builder-config"
+    # Quarkus extensions required for workflows persistence. These extensions are used by the SonataFlow build system,
+    # in cases where the workflow being built has configured postgresql persistence.
+    postgreSQLPersistenceExtensions:
+      - groupId: io.quarkus
+        artifactId: quarkus-jdbc-postgresql
+        version: 3.2.10.Final
```

Comment:
imho this should be globally configurable at single place across the repository

Reply:
Hi  @domhanak , this information is already configured in one single place, here:  https://github.com/apache/incubator-kie-kogito-serverless-operator/pull/457/files#diff-ddac596008b9e8c9968a61666b661f894cd11b277ac2515a59f042844413f15b, in the file **kogito-serverless-operator/config/manager/controllers_cfg.yaml** which is correct.

Then, the reason why you can see this information in the files: **sonataflow-operator-controllers-config_v1_configmap.yaml** and **operator.yaml**, is because this files are automatically generated, and these files are necessary for the operator deployment, like the other manifests.

But, the source of true for these generation is the file **controllers_cfg.yaml**.

Reply:
Thanks for confirmation